static/js/featuredialog.js

```diff
@@ -0,0 +1,45 @@
+var FeatureDialog = Backbone.View.extend({
+	className: "marker-dialog",
+	events: {
+		"submit" : "submit",
+		"click .save-button" : "submit",
+		"click .close-button" : "close"
+	},
+	initialize: function(options) {
+		this.markers = options.markers;
+		this.type = options.type;
+		this.event = options.event;
+
+	},
+	render: function() {
+		this.$el.append($("#feature-dialog-template").html());
+		$(document.body).append(this.$el);
+		this.modal = this.$el.find(".modal");
```

Comment:
If you use the `id` of the modal in the `href`, you won't need to call `modal("show")`, because it will happen by itself